opentreemap/otm_comments/tests.py

```diff
@@ -155,7 +155,7 @@ def test_sorting(self):
 
 
 def _comment_ids_to_params(*args):
-    return {'comment-ids': ','.join(str(arg) for arg in args)}
+    return {'ids': ','.join(str(arg) for arg in args)}
```

Comment:
I changed this to `ids` because the JS that calls this view function is now shared by photo and comment moderation, so it didn't make sense to call it `comment-ids` any more.